pipeline_dsl/concourse/pipeline.py

```diff
@@ -35,7 +36,8 @@ def __init__(self, name, image_resource={"type": "registry-image", "source": {"r
                 self.init_dirs[os.path.basename(dir)] = dir
         else:
             for key, script in script_dirs.items():
-                self.init_dirs[key] = os.path.abspath(os.path.join(dirname, script))
```

Comment:
Isn't it a bit confusing that glob only work if you do not a use a list? Is there a reason?

Reply:
You are right. I should implement that as well. But I didn't implement it because we haven't decided if we want to go with the globbing approach.

Reply:
I think we can go that way. AFAIK, I was the only one promoting the other way, right :) The only think is, that this approach is not compatible, so we would have to change our pipelines. Otherwise, we might end up with binaries in the pipeline, right?

Reply:
If implemented globbing for script_dirs list in https://github.com/SAP/pipeline-dsl/pull/17/commits/36b7efc1b1e2c48a5fe9dff0fe5ede0b016ca58b.
As there is no option for the user to specify a key, the key will be created by:
* using the dirname if no specific/direct file is specified (e.g. glob or directory)
  * Example: `test` for `test/*.yaml` or `test` for `test/`
* using the specified file as key (e.g. `test.yaml`)
  * Example: `test.yaml` for `test.yaml` or `test/test.yaml' for `test/test.yaml`